tox.ini

```diff
@@ -13,14 +13,11 @@
 #  and also to help confirm pull requests to this project.
 
 [tox]
+minversion = 3.7.0
 envlist = py{27,35,36,37}
+isolated_build = true
```

Comment:
And again these are unrelated changes - please limit the PR to achieving one thing (the addition of `pyproject.toml`)

Reply:
This one actually is related - this makes `tox` use PEP 517, which I think is important here.

Reply:
Hmm, OK. At a minimum, this needs a comment.  In spite of the comment at the top of `tox.ini`, I suspect people will use this file as a model, and so why this is needed should be explained.